fedimint-api/src/lib.rs

```diff
@@ -164,37 +197,9 @@ impl From<PeerId> for u16 {
     }
 }
 
-impl Amount {
```

Comment:
I didn't notice these existed because this `impl` block was so far away from `struct Amount`. :shrug:

Reply:
Something I wanted to ask for a long time: what's a good standard for ordering impl blocks? I'm kinda switching between the two:

* Struct1, Impl1, Trait1Impl1, Trait2Impl1, Struct2, Impl2
* Struct1, Struct2, Impl1, Trait1Impl1, Trait2Impl1, Impl2

Can `rustfmt` automatically enforce some ordering?

Reply:
> Generally I try to keep impls right after their struct/trait definition, but I never thought about it too much. Seemed most natural to have the close when possible.

+1 to the natural feel of this ordering.

Within the same file, I'd wish for a `rustfmt` enforce of `Struct1, Impl1, Trait1Impl1, Trait2Impl1, Struct2, Impl2` - with `Struct1` being a more prominent definition relative to `Struct2` etc. and similarly for the traits, so the most important Structs and Traits are top of the module